src/components/createTestCase/CreateTestCase.test.tsx

```diff
@@ -86,6 +86,15 @@ jest.mock("@madie/madie-util", () => ({
     getAccessToken: () => "test.jwt",
     getUserName: () => MEASURE_CREATEDBY,
   }),
+  routeHandlerStore: {
+    subscribe: (set) => {
+      // set(measure)
```

Comment:
What does this line do? Should it be commented out?

Reply:
removed, was brought in from copy/paste